primitives/types/dispatch_result_test.go

```diff
@@ -0,0 +1,63 @@
+package types
+
+import (
+	"testing"
+
+	sc "github.com/LimeChain/goscale"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	postDispatchInfo2 = PostDispatchInfo{
+		ActualWeight: sc.NewOption[Weight](WeightFromParts(1, 2)),
+		PaysFee:      PaysYes,
+	}
```

Comment:
You can use `postDispatchInfo`

Reply:
It is the same

Reply:
yes, but I prefer to use something that is declared in the same file